src/plugins/cal-events/cal-events.js

```diff
@@ -436,6 +458,10 @@ $document.on( "keydown", ".wb-calevt-cal .cal-days td > ul li", function( event
 	}
 } );
 
+$document.on( "focusout", ".wb-calevt-cal td > ul", function( event ) {
```

Comment:
Let's take the opportunity to optimize the code and reduce number of duplicata

```suggestion
$document.on( "focusout", selectorEvent + ". td > ul", function( event ) {
```

At the top of the plugin, declare that variable near line 18 https://github.com/wet-boew/wet-boew/blob/master/src/plugins/cal-events/cal-events.js#L18
```
var componentName = "wb-calevt",
	selector = "." + componentName,
	componentEventName = componentName + "-cal",
	selectorEvent = "." + componentEventName,
	initEvent = "wb-init" + selector,
```

And replace line 91 https://github.com/wet-boew/wet-boew/blob/master/src/plugins/cal-events/cal-events.js#L91

```
		$container = $( "#" + containerId ).addClass( componentEventName );
```

And replace the other event that rely on an hard coded selector like: `.wb-calevt-cal`